### DISCUSSION TITLE
Audio equaliser api integration

### DISCUSSION THREAD

**Discussion by swapnildabhade** [wwmm/easyeffects#2456] [Q&A] [answered]
Can we get API call for the audio equaliser to integrate it with another cpp program?
I am working on a command line based tool to apply filter on audio file. 
Is any example available?

**Comment by wwmm** [comment, marked as answer]
If what you want is to change EasyEffects parameters from a third party tool it is possible through the same procedure described for PulseEffects at #916. Any program capable of using `GSettings` API can change EasyEffects settings. You just have to open dconf-editor and take a look at the correct key paths.
